src/SFA.DAS.ApprenticeApp.Domain/Models/MyApprenticeship.cs

```diff
@@ -25,6 +25,12 @@ public TimeSpan? ApprenticeshipLength
                 return null;
             }
         }
+        public ApprenticeshipType ApprenticeshipType { get; set; }
+    }
 
+    public enum ApprenticeshipType
```

Comment:
Please use enum is description attributes and reference them in code in order to not have to do string replacement